packages/keystatic/src/component-blocks/cloud-image-preview.tsx

```diff
@@ -418,6 +414,10 @@ function imageWithTransforms(options: {
 }) {
   let { fit = 'scale-down', source, height, width } = options;
 
+  if (!/https?:\/\/[^\.].keystatic.net/.test(source)) {
```

Comment:
This would make testing and custom domains quite impractical so i'm not very keen on it? separately, it wouldn't ever work right:
```suggestion
  if (!/https?:\/\/[^\.]+.keystatic.net/.test(source)) {
```

Reply:
Yeah, good catch on the regex, thanks

Agreed, was thinking about custom domains when I write it and I've got ideas on how we can make this better later (before we get to custom domains even) but for now it was causing a 404 on images from a different service so I think it's worth putting in for now.